cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```
@@ -47,6 +47,8 @@ spec:
   crashCollector:
     disable: false
   storage:
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement.
```

I would suggest something like:

```suggestion
    #  onlyApplySpecificOSDPlacement  will only use `placement` or  `preparePlacement` under `storageClassDeviceSets` when set to true.  In case of false,  `placement.All` will be merged with `placement` or  `preparePlacement` under `storageClassDeviceSets`
```